### PR TITLE
Update SSAARenderPass.js

### PR DESCRIPTION
**Pull request by moshuying**
Fix file references

Related issue: #XXXX

**Description**

A clear and concise description of what the problem was and how this pull request solves it.

This contribution is funded by [Example](https://example.com).

### CODE REVIEW

File: examples/jsm/postprocessing/SSAARenderPass.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ import {
 	UniformsUtils,
 	WebGLRenderTarget
 } from '../../../build/three.module.js';
-import { Pass, FullScreenQuad } from '../postprocessing/Pass.js';
```
Comment (Mugen87) [inline examples/jsm/postprocessing/SSAARenderPass.js:10]: This path is actually a leftover from the time when modules were generated from `examples/js`. It seems almost all post processing pass classes use `../postprocessing/Pass.js;`right now.

Do you mind updating the other classes in `examples/jsm/postprocessing`, too?

Comment (Mugen87) [reply examples/jsm/postprocessing/SSAARenderPass.js:10]: If you don't want to update the other passes, that's fine. We can still merge the PR and I do a clean up afterwards 👍 .

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

File: examples/jsm/postprocessing/SSAARenderPass.js (hunk 1 above)
Comment (moshuying) [reply examples/jsm/postprocessing/SSAARenderPass.js:10]: just this file😂,Thank you for your reply. I found that many files have this path problem. It is not very convenient for me to deal with the content of the file code, but it does not affect the operation at all。

Comment (Mugen87) [reply examples/jsm/postprocessing/SSAARenderPass.js:10]: #22832

**Pull request merged**